docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,20 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+By default, Amazon DynamoDB is available for all AWS users. You do not need to configure anything manually.
+Scalar schema loader [helm charts](https://github.com/scalar-labs/helm-charts/tree/main/charts/schema-loading) will help you to configure DynamoDB.
```

Comment:
Is it possible to create the schema for Scalar DB deployment?
I think it is possible to create a Scalar DL schema.

For Scalar DB, we need to use [Scalar DB Releases](https://github.com/scalar-labs/scalardb/releases) or something

Reply:
Thank you.
I will update it more clearly for both scalar DB and DL.